.github/workflows/os-drift-nightly.yml

```
@@ -0,0 +1,338 @@
+        codename: [trixie, bookworm, bullseye]
+    # Run all steps directly inside the unpinned debian:${codename} image so
+    # the apt-cache and pip resolution checks exercise the same package set
+    # install.sh would see on that OS base. The :latest tag is intentional —
+    # we WANT to fail if Debian ships a breaking change.
+    container:
+      image: debian:${{ matrix.codename }}
+      options: --user root
+    # Allow a single matrix leg to be skipped by the workflow_dispatch input.
+    if: |
+      github.event_name != 'workflow_dispatch' ||
+      github.event.inputs.codename == '' ||
```

actionlint (1.7.12)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)